StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentElement.swift

```diff
@@ -155,6 +170,8 @@ public final class EmbeddedPaymentElement {
             self.intent = loadResult.intent
             self.embeddedPaymentMethodsView = embeddedPaymentMethodsView
             self.containerView.updateEmbeddedPaymentMethodsView(embeddedPaymentMethodsView)
+            self.formViewController = formViewController
+            self.formCache = .init()
```

Comment:
I'm not following why this class needs a formCache. Seems like only `formViewController` would need a copy since we aren't preserving the cache between update calls?

Reply:
The cache needs to be kept in storage between creations (presentations) of formViewController, to preserve input of multiple PMs, and partial user input. That way if we
1. Present Klarna enter email
2. Close Klarna form
3. Open card, start typing a number
4. Close card
5. Open Klarna, we still have the cached input in Klarna email field.